packages/vats/src/vat-bank.js

```diff
@@ -319,6 +331,10 @@ export function buildRootObject() {
 
         /**
          * Add an asset to the bank, and publish it to the subscriptions.
+         * If nameAdmin is defined, update with denom to AssetInfo entry.
+         *
+         * Note that AssetInfo has the settled identity of the issuer,
```

Comment:
good doc. for my understanding, is that the same as saying "has a **presence** of the issuer"?

Reply:
yes, unless the issuer was actually created in the publishing vat, in which case it would be the object itself and not a presence for it.